src/main/scala/beam/agentsim/agents/ridehail/AlonsoMoraPoolingAlgForRideHail.scala

```diff
@@ -339,7 +340,8 @@ object AlonsoMoraPoolingAlgForRideHail {
                 Trip(theActivity, None, null),
                 BeamMode.RIDE_HAIL,
                 Relocation,
-                leg.startTime
+                leg.startTime,
+                leg.startTime + Int.MaxValue - 30000000
```

Comment:
why 30000000? seems very arbitrary and confusing... it's roughly 346 days, is that meaningful?